pages/common/subl.md

```diff
@@ -0,0 +1,19 @@
+# subl
+
+> Sublime Text, a sophisticated text editor for code, markup and prose.
+
+- Open a file or folder:
+
+`subl {{path/to/file_or_folder}}`
+
+- Open a file or folder in a new window:
+
+`subl -n {{path/to/file_or_folder}}`
```

Comment:
Is there a long form of this argument at all? We do generally prefer them if they are available. Same for the other examples below.

Reply:
Seems fine to me. Since there is a word (new) whose first letter matches the argument. Same for -a -> Add.

Reply:
True, @agnivade